common/lib/xmodule/xmodule/course_module.py

```diff
@@ -196,6 +196,13 @@ class CourseFields(object):
         help=_("Date that certificates become available to learners"),
         scope=Scope.content
     )
+
+    archiving_lms_date = Date(
```

Comment:
can we please change this to be archive_course_in_lms? little bit more discriptive and actualy states the course will be archived, users might just interpret this wrongly.

Reply:
Cool, will do.